recipes-core/packagegroups/packagegroup-ni-internal-deps.bb

```diff
@@ -33,3 +33,8 @@ RDEPENDS_${PN} += "\
 RDEPENDS_${PN}_append_x64 = "\
 	qtbase \
 "
+
+# ni-rdma, libnirdma
```

Comment:
Can you add a technical contact here - someone who can speak to the dependency?

Reply:
Sure. @ericgross2 just approved this review so I'm taking that as him volunteering to be that contact.

Reply:
added in v2